reana_job_controller/kubernetes_job_manager.py

```diff
@@ -511,6 +511,12 @@ def _add_rucio_init_container(self, secrets_volume_mount, secret_env_vars):
 
         rucio_account = os.environ.get("RUCIO_USERNAME", "")
         voms_proxy_vo = os.environ.get("VONAME", "")
+        rucio_host = os.environ.get(
+            "RUCIO_CFG_RUCIO_HOST", f"https://{voms_proxy_vo}-rucio.cern.ch"
```

Comment:
Cosmetics: we currently expose the following Rucio-related environment variables to users:

- VONAME
- VOMSPROXY_FILE
- RUCIO_USERNAME

documented in https://docs.reana.io/advanced-usage/access-control/rucio/#uploading-secrets

I.e. the username is just `RUCIO_USERNAME`, not `RUCIO_CFG_USERNAME` as used in the j2 template.

Hence, to be more consistent with what we have for USERNAME, what about naming the two new variables simply:

- RUCIO_RUCIO_HOST
- RUCIO_AUTH_HOST

WDYT?

Reply:
Otherwise the PR works well :+1:  I had tested a "good" auth host vs "bad" auth host.

BTW one minor observation is that if a user has access to two Rucio instances, and would like to use both during the same workflow e.g. different Rucio instance in different workflow steps, then this is currently not easily possible due to using common environment variable secrets..

If this use case might come in your VRE installation, then we should rather consider switching to using workflows hints, such as:

```yaml
      - name: stagein
        environment: 'docker.io/reanahub/reana-auth-rucio:1.1.0'
        voms_proxy: true
        rucio: true
        rucio_auth_host: example.org 
        commands:
          - rucio download simko_test:helloworld --dir data
```

Something that could push for later if you don't have an immediate use case for this?

Reply:
Concerning the variables exposed to users, I like RUCIO_RUCIO_HOST and RUCIO_AUTH_HOST. 
Regarding accessing multiple instances in different steps is a very useful thing, and could be very beneficial in the future, let's use this first implementation and work on the second one in a second step.

Reply:
Thanks, I'll squash and rebase whilst merging